src/app/not-found.tsx

```
@@ -0,0 +1,27 @@
+import { Link } from "next-view-transitions"
+import * as React from "react"
+
+interface Props {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)